sdk/storage/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.12)
+
+project (azure-storage LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+option(BUILD_STORAGE_SAMPLES "Build sample codes" ON)
+
+set(AZURE_STORAGE_HEADER
+    inc/blobs/blob.hpp
+    inc/blobs/internal/protocol/blob_rest_client.hpp
+    inc/common/internal/protocol/rest_client_utility.hpp
+    inc/common/xml_wrapper.hpp
+)
+
+set(AZURE_STORAGE_SOURCE
+    src/blobs/internal/protocol/blob_rest_client.cpp
+    src/common/internal/protocol/rest_client_utility.cpp
+    src/common/xml_wrapper.cpp
+)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
```

Comment:
isn't this the default? 
Why do we need to set this?

Reply:
This is done to other Storage SDK's CMake file and I did not doublethink on this. I'll remove them for now.